### PR TITLE
Satisfy ansible by updating general.yml

### PR DESCRIPTION
**Pull request by jhadvig**
Backport changes of inventory's general.yml from [aos-cd-jobs](https://github.com/openshift/aos-cd-jobs/commit/c4cd471f9b469826fa262a74c0914263d10e3ce8) repo

### CODE REVIEW

**Comment by enj** [comment]
Please no politics in commit messages.  I get enough of it during lunch.

**Comment by stevekuznetsov** [comment]
@mfojtik @enj how do you see this stuff before I do?

File: oct/ansible/oct/roles/inventory/files/general.yml [hunk 1]
```diff
@@ -8,4 +7,6 @@ openshift_master_identity_providers:
   - name: "allow_all"
     login: "true"
     challenge: "true"
-    kind: "AllowAllPasswordIdentityProvider"
\ No newline at end of file
+    kind: "AllowAllPasswordIdentityProvider"
+deployment_type: "origin"
+ansible_ssh_user: "ec2-user"
```
Comment (stevekuznetsov) [inline oct/ansible/oct/roles/inventory/files/general.yml]: We won't want this guy in here -- it lives in the `host_vars` for this inventory.

**Comment by jhadvig** [comment]
@stevekuznetsov updated :)

**Pull request merged**